src/Service/S3/tests/Unit/Input/PutObjectRequestTest.php

```diff
@@ -7,7 +7,7 @@
 
 class PutObjectRequestTest extends TestCase
 {
-    public function testRequestBody()
+    public function testRequest()
     {
         $bucket = 'foo';
         $key = 'bar/baz/biz';
```

Comment:
These variables are only used once. Are they auto generated?

Reply:
nop, did you try `git blame`?..

Reply:
They were used twice but this PR removed once instance. 

Im fine with this, I just wanted to make sure we dont generate variables that are only used once.

Reply:
ohh. my bad. Removed